### ISSUE TITLE
RailsCharts::Javascript.new is evaluated only once

### ISSUE THREAD

**Issue by deepracercve** [railsjazz/rails_charts#10]
In `custom_chart` - `RailsCharts::Javascript.new` is evaluated once.

only `valueFormatter` is available in the generated code. `position` is not.

```
              tooltip: {
                trigger: 'axis',
                valueFormatter: RailsCharts::Javascript.new("(value) => '$' + value"),
                position: RailsCharts::Javascript.new("function (pt) { return ['90%', '10%']; }")
              },
              ...
```

**Comment by deepracercve** [comment]
OK - will try

**Comment by deepracercve** [comment]
I believe the offending line is this https://github.com/railsjazz/rails_charts/blob/a80c2aac10bc95baf565f74d7d9d9e5cf3b69d7d/lib/rails_charts/base_chart.rb#L85

`gsub!` might be doing greedy lookup - needs to be changed.

**Comment by manastyretskyi** [comment]
I've pushed new version to rubygems.
Thank you again for reporting and fixing this 🙏

**Comment by igorkasyanchuk** [comment]
@tamersalama please share your experience of using this gem for charts. is it convenient? 

PS: we have more nice gems :) [rails_live_reload](https://github.com/railsjazz/rails_live_reload) might be extremely helpful for you

**Comment by tamersalama** [comment]
Thank you for creating this project. It is quite convenient. And the `custom_chart` give you super-powers :). 
I was wondering if there would be more flexibility in setting the data - instead of passing it as a first argument. But that's another story :)
